control-station/tests/test_control_station.py

```diff
@@ -0,0 +1,16 @@
+from unittest import Mock
+from control_station import ControlStation
+
+class TestControlStation():
+    def _make_testee(self):
+        testee = ControlStation()
+        return testee
+
+    def test_read_timeout(self):
+        pass
+
+    def test_context_manager_usage(self):
+        pass
+
+    def test_repl(self):
+        pass
```

Comment:
I don't know much about unit testing, so I'm going to trust this and assume the "pass" will be filled out later. Why import Mock though?